utility/src/main/java/org/oppia/util/parser/GlideImageLoader.kt

```diff
@@ -27,4 +31,21 @@ class GlideImageLoader @Inject constructor(
       .load(model)
       .into(target)
   }
+
+  override fun loadSvg(imageUrl: String, target: CustomTarget<PictureDrawable>) {
+    val model: Any = if (cacheAssetsLocally) {
+      object : ImageAssetFetcher {
+        override fun fetchImage(): ByteArray = assetRepository.loadRemoteBinaryAsset(imageUrl)()
+
+        override fun getImageIdentifier(): String = imageUrl
+      }
+    } else imageUrl
+
+    Glide.with(context)
+      .`as`(PictureDrawable::class.java)
+      .fitCenter()
+      .apply(RequestOptions.diskCacheStrategyOf(DiskCacheStrategy.NONE))
```

Comment:
Reminder to please add a TODO here for #45 (see my earlier comment).

Reply:
Done.